src/sestest/NodeEntitlementsBuilder.cs

```diff
@@ -152,19 +171,13 @@ private IEnumerable<Errorable<IPAddress>> Addresses()
             {
                 foreach (var address in _commandLine.Addresses)
                 {
-                    if (IPAddress.TryParse(address, out var ip))
-                    {
-                        yield return Errorable.Success(ip);
-                    }
-                    else
-                    {
-                        yield return Errorable.Failure<IPAddress>($"IP address '{address}' not in expected format (IPv4 and IPv6 supported).");
-                    }
+                    yield return TryParseIPAddress(address);
                 }
 
                 yield break;
             }
 
+            // No IP addresses specified by the user, default to using all from the current machine
             foreach (var i in NetworkInterface.GetAllNetworkInterfaces())
```

Comment:
how about making it a method "ListAllIPAddresses"?

Reply:
Refactored to use new method `ListMachineIpAddresses()`.